cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -255,6 +255,12 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, k8sVersion *versio
 		} else {
 			command = append(command, fmt.Sprintf("--audit-log-maxage=%d", *cfg.AuditPolicyConfiguration.LogMaxAge))
 		}
+		if cfg.AuditPolicyConfiguration.WebhookConfigPath != nil {
+			command = append(command, "--audit-webhook-config-file="+kubeadmconstants.GetStaticPodAuditWebhookConfigFile())
```

Comment:
Please add to the test cases or create new test cases for the api server params

Reply:
Will do.